### ISSUE TITLE
Hero image has no alt-text

### ISSUE THREAD

**Issue by erikkroes** [electron/electronjs.org#4582]
There is a large SVG at the top of the page in a div named "hero". It does not have an accessible name, but is also not marked up as decorative. From the context I would assume it is decorative. This can be fixed by adding aria-hidden to hide it from Assistive Technology.

More information about the attribute can be found here: https://www.w3.org/TR/wai-aria-1.1/#aria-hidden

===

This finding of this issue has been recorded. It is part of my weekly accessibility stream on Twitch: twitch.tv/erikKroes
A recording should be up soon at youtube: https://www.youtube.com/channel/UCwrpGO76k2HyUQspl-rG2eQ/videos

**Comment by tt-le** [comment]
Hi @erickzhao, can I be assigned to this issue? Thanks!
